lang_scala/parsing/Token_helpers_scala.ml

```diff
@@ -246,6 +246,13 @@ let isIdent = function
 
   | _ -> None
 
+let isMetavar = function
```

Comment:
You can use AST_generic_.is_metavar_name.
You can look at the toplevel comment in this file to understand why it's called AST_generic_
with the suffix underscore.